tensorbay/cli/utility.py

```diff
@@ -387,3 +395,38 @@ def _set_request_config(config_parser: ConfigParser) -> None:
             client_config.is_internal = config_section.getboolean("is_internal")
         if "max_retries" in config_section:
             client_config.max_retries = config_section.getint("max_retries")
+
+
+def echo_response(err: InternalServerError) -> None:
+    """Log request and response when InternalServerError happens.
+
+    Arguments:
+        err: The InternalServerError raised from the CLI.
+
+    """
+    if logger.disabled:
+        click.echo(dump_request_and_response(err.response))
+
+
+def exception_decorator(cli_func: _Callable) -> Any:
```

Comment:
the return type should be `_Callable` to tell mypy the decorator will not change the function signature.

Reply:
`cli_func` -> `func`
`exception_decorator` -> `exception_handler`?